src/diffusers/pipelines/audioldm2/pipeline_audioldm2.py

```diff
@@ -51,19 +51,32 @@
 EXAMPLE_DOC_STRING = """
     Examples:
         ```py
-        >>> from diffusers import AudioLDM2Pipeline
-        >>> import torch
         >>> import scipy
+        >>> import torch
+        >>> from diffusers import AudioLDM2Pipeline
 
         >>> repo_id = "cvssp/audioldm2"
         >>> pipe = AudioLDM2Pipeline.from_pretrained(repo_id, torch_dtype=torch.float16)
         >>> pipe = pipe.to("cuda")
 
-        >>> prompt = "Techno music with a strong, upbeat tempo and high melodic riffs"
-        >>> audio = pipe(prompt, num_inference_steps=200, audio_length_in_s=10.0).audios[0]
+        >>> # define the prompts
+        >>> prompt = "The sound of a hammer hitting a wooden surface."
+        >>> negative_prompt = "Low quality."
 
-        >>> # save the audio sample as a .wav file
-        >>> scipy.io.wavfile.write("techno.wav", rate=16000, data=audio)
+        >>> # set the seed
+        >>> generator = torch.Generator("cuda").manual_seed(0)
+
+        >>> # run the generation
+        >>> audio = pipe(
+        ...     prompt,
+        ...     negative_prompt=negative_prompt,
+        ...     num_inference_steps=200,
+        ...     audio_length_in_s=10.0,
+        ...     num_waveforms_per_prompt=3,
```

Comment:
Let's maybe pass the generator if we define it
```suggestion
        ...     num_waveforms_per_prompt=3,
        ...     generator=generator,
```